src/goose/synopsis/synopsis.md

```diff
@@ -0,0 +1,25 @@
+# Current System Info
+
+{{system.info()}}
+
+# Hints
+
+{{synopsis.hints}}
+
+# Relevant Files
```

Comment:
this is really neat! i am guessing the context mgmt work can be injected here

Reply:
yeah I think thats a great home for it!